Python/compile.c

```diff
@@ -5371,38 +5371,28 @@ typedef struct {
     PyObject *temp_symbols;
     PyObject *fast_hidden;
     jump_target_label cleanup;
-    jump_target_label end;
 } inlined_comprehension_state;
 
 static int
-push_inlined_comprehension_state(struct compiler *c, location loc,
-                                 PySTEntryObject *entry,
-                                 inlined_comprehension_state *state)
+compiler_tweak_inlined_comprehension_scopes(struct compiler *c, location loc,
```

Comment:
I'm not sure what this does, maybe change "tweak" to something more informative?

Can we also add a comment as we why we need to "tweak" the "state", rather than creating a new symbol table entry for the comprehension?

Reply:
I have this issue to look into this area - let's discuss there. https://github.com/python/cpython/issues/122985

Reply:
(Sorry, I already clicked auto-merge before your comment came through).